src/utils.ts

```diff
@@ -70,6 +75,8 @@ const converted = (s: string): string => {
     return s;
 };
 
+const isSlug = (str: string) => str === slugify(str);
```

Comment:
Note: Add return type of `boolean` and maybe add `isString` functionality?

Reply:
@rmi22186 I'm torn on this. Do we explicitly need to check if something is both a slug and a string, or can we just simplify things and force slug and string checks in one place?

Reply:
Are these comments outdated? you have `isString` on line 78 and a separate `isSlug` method with no `isString` check inside of it, so I'm a bit confused.

Reply:
if it matters, when you call `slugify(8)`, it throws an error as opposed to logging an error